shacl12-test-suite/tests/core/node/memberShape-001.ttl

```diff
@@ -0,0 +1,102 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://datashapes.org/sh/tests/core/node/memberShape-001.test#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:IRIListShape
+  rdf:type sh:NodeShape ;
+  sh:memberShape [ sh:nodeKind sh:IRI ] ;
+  sh:targetNode ( ex:Alice ex:Bob ) ;
+  sh:targetNode () ;
```

Comment:
This target isn't among the expected test results, but looks like it should trigger a violation.

Reply:
This is a valid SHACL list (it is just `rdf:nil`) so I expect this to pass.

I can see cases where you'd want to make sure the list does have have some elements - I've raised https://github.com/w3c/data-shapes/issues/414 to propose `sh:minCollectionLength` and `sh:maxCollectionLength` as constraints